.github/workflows/version-bump.yml

```diff
@@ -72,7 +72,9 @@ jobs:
           fi
           git checkout -b topic/v${{ steps.set_release_name.outputs.release_version }}
       - name: Run backstage-cli versions:bump --release ${{ inputs.release_line || 'next' }} command
-        run: yarn backstage-cli versions:bump --release ${{ inputs.release_line || 'next' }}
+        run: |
+          export YARN_ENABLE_IMMUTABLE_INSTALLS=false 
```

Comment:
You can set this with an `env` key on the step rather than using export, a tiny bit nicer 😁

Reply:
I think it would just be like

```yaml
      - name: Run backstage-cli versions:bump --release ${{ inputs.release_line || 'next' }} command
        run: yarn backstage-cli versions:bump --release ${{ inputs.release_line || 'next' }}
        env:
          YARN_ENABLE_IMMUTABLE_INSTALLS: false
```

Reply:
Awesome, push an update with this just now